### PR TITLE
Cross site scripting

### PR DESCRIPTION
**Pull request by tbetley**
Major:
- Add cross site scripting vulnerability.

Minor:
- Fix bug where admin page link was not showing for admin on secure version of site.

### CODE REVIEW

File: Website/views/pages/admin.ejs [hunk 1]
```diff
@@ -21,19 +21,32 @@
...
             </thead>
             <% for(var i=0; i < data.length; i++) { %>
-                <tr>
-                    <td><%= data[i].account_id %></td>
-                    <td><%= data[i].user_id %></td>
-                    <td><%= data[i].routing_number %></td>
-                    <td><%= data[i].account_number %></td>
-                    <td><%= data[i].balance %></td>
-                </tr>
+                <% if(secure) { %>
```
Comment (martinky24) [inline Website/views/pages/admin.ejs:31]: What's the difference here? Am I overlooking something or are the two branches the same?

File: Website/views/partials/header.ejs [hunk 2]
```diff
@@ -30,7 +30,7 @@ if(!secure){insecureCss="background-color: red!important;"}
                 <a class="nav-link" href="/logout">Logout</a>
             </li>
         </ul>
-        <% if (secure && (username == "admin")) { %>
+        <% if (username == "admin") { %>
```
Comment (martinky24) [inline Website/views/partials/header.ejs:33]: I just fixed this same bug in my other branch 😅 glad we both caught it

**Review by martinky24 — APPROVED** [review]
Overall very simple and very sharp. Aside from the potentially duplicated content in the `if` statement this seems good to add.

File: Website/views/pages/admin.ejs (hunk 1 above)
Comment (tbetley) [reply Website/views/pages/admin.ejs:31]: EJS has built in XSS measures. The "<%=" syntax encodes anything after it, preventing a malicious script from executing. The "<%-" syntax does not encode anything, so it's up to the browser to render it, which will execute scripts.

Comment (martinky24) [reply Website/views/pages/admin.ejs:31]: Didn't notice the "=" vs the "-" difference, oops. Thanks!

**Review by nstapler — APPROVED** [review]
LGTM

**Pull request merged**